index.js

```diff
@@ -49,28 +50,37 @@ client.on("message", function (message) {
     inventoryCommand(message);
   } else if (message.content.startsWith("!lookup")) {
     lookUpCommand(message);
+  } else if (message.content.startsWith("!remove")) {
+    removeCardCommand(message);
   }
 });
 
 const drawCommand = (message) => {
-  var ref = db.ref("users");
+  const ref = db.ref("users");
   ref.once("value", function (snapshot) {
     const userData = snapshot.val();
-    let userDraw = message.author.id;
-    let user = userData[userDraw];
+    const userID = message.author.id;
+    const user = userData[userID];
     const userDoesNotExist = !user;
     let timeDifference;
     if (user) {
       let lastBotUse = user.lastDrawTime;
       timeDifference = Date.now() - lastBotUse;
-      console.log(userData);
-      console.log(lastBotUse);
-      console.log(Date.now());
-      console.log(timeDifference);
+      const userInventory = userData[userID].inventory;
+      const inventorySize = Object.keys(userInventory).length;
+      const maxNumOfCards = 60;
```

Comment:
So in JS/Java, constants that are specifically hardcoded numbers, strings, etc (as opposed to variables that just happen not to change) are usually coded as either `ALL_CAPS` or `PascalCase`. I prefer the former, so this might be `MAX_NUM_OF_CARDS`.

Oftentimes you'll find there is an existing natural name for constants like this (e.g. `ONE_MINUTE`, `CELSIUS_TO_FAHRENHEIT_RATIO`, etc). Funnily enough the one for this was used in the Magic Number wiki page example: 
![image](https://user-images.githubusercontent.com/101266/98888571-389cdb00-2466-11eb-90e2-7944195fed04.png) (e.g. deck size, or deck limit)